substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -415,26 +415,29 @@ def help_stdout_check(output):
         if t:
             hellomodule(args.extra_image_builder_arguments)
 
-    with Task('Validate JSON build output', tasks, tags=[mx_gate.Tags.style]) as t:
+    with Task('Validate JSON build info', tasks, tags=[mx_gate.Tags.style]) as t:
         if t:
             import json
             try:
                 from jsonschema import validate as json_validate
                 from jsonschema.exceptions import ValidationError, SchemaError
             except ImportError:
                 mx.abort('Unable to import jsonschema')
+            with open(join(suite.dir, '..', 'docs', 'reference-manual', 'native-image', 'assets', 'build-artifacts-schema-v0.9.0.json')) as f:
+                build_artifacts_schema = json.load(f)
             with open(join(suite.dir, '..', 'docs', 'reference-manual', 'native-image', 'assets', 'build-output-schema-v0.9.1.json')) as f:
-                json_schema = json.load(f)
+                build_output_schema = json.load(f)
             with tempfile.NamedTemporaryFile(prefix='build_json') as json_file:
-                helloworld(['--output-path', svmbuild_dir(), f'-H:BuildOutputJSONFile={json_file.name}'])
+                helloworld(['--output-path', svmbuild_dir(), f'-H:BuildOutputJSONFile={json_file.name}', f'-H:BuildArtifactsJSON={svmbuild_dir()}'])
```

Comment:
Isn't this option no longer valid? `-H:+BuildArtifacts`?

This is also the reason for the gate failure it seems:
```
 Running: mx --primary-suite-path substratevm --java-home=/home/runner/work/graal/graal/jdk helloworld --output-path /home/runner/work/graal/graal/substratevm/svmbuild -H:BuildOutputJSONFile=/tmp/build_jsontyc_wdu_ -H:BuildArtifactsJSON=/home/runner/work/graal/graal/substratevm/svmbuild
Error: Could not find option 'BuildArtifactsJSON'. Did you mean one of these: BuildArtifacts. Use -H:PrintFlags= to list all available options.
```

Reply:
Ype, forgot to update that test when I pushed this. Should be fixed now.